test/Sentry.Tests/SentryClientTests.cs

```diff
@@ -55,7 +54,7 @@ public void CaptureEvent_IdReturnedToString_NoDashes()
 
             var actual = sut.CaptureEvent(evt);
 
-            var hasDashes = actual.ToString().Contains('-');
+            var hasDashes = actual.ToString().Contains("-");
```

Comment:
This previously used LINQ on net462 but not on netcore. Changed to string so it uses intrinsic string method in both cases.